OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/SecurityTokenInfo.java

```diff
@@ -104,20 +107,21 @@ public static SecurityTokenInfo newInstanceDebugLockedHard() {
             TokenType.NITROKEY_STORAGE,
             TokenType.GNUK_OLD,
             TokenType.GNUK_UNKNOWN,
-            TokenType.GNUK_NEWER_1_25
+            TokenType.GNUK_1_25_AND_NEWER
     ));
 
     private static final HashSet<TokenType> SUPPORTED_USB_RESET = new HashSet<>(Arrays.asList(
             TokenType.YUBIKEY_NEO,
             TokenType.YUBIKEY_4,
             TokenType.NITROKEY_PRO,
-            TokenType.GNUK_NEWER_1_25
+            TokenType.GNUK_1_25_AND_NEWER
```

Comment:
reset and put key are now the same set of devices. do we expect this to ever change again? Hopefully not? :)